app/Enums/MediaType.php

```diff
@@ -38,21 +41,54 @@ public function prefix(): string
     public function isInstantlyAvailable(): bool
     {
         return match ($this) {
-            self::IMAGE => true,
+            self::IMAGE,
+            self::DOCUMENT => true,
             self::VIDEO => false
         };
     }
 
     /**
-     * Get whether this media needs a short invalidation path for the CDN.
+     * Get whether this media needs exhaustive invalidation for the CDN:
```

Comment:
needsExhaustiveCdnInvalidation. helpful explanation:  needs exhaustive invalidation for the CDN...